### DISCUSSION TITLE
How to use utility classes from global CSS inside custom elements?

### DISCUSSION THREAD

**Discussion by callumlocke** [lit/lit#3460] [Q&A]
New to Lit and experimenting. I'm trying to make an app using a few custom elements and some global styles.

Let's say I have a **global** CSS rule like `.debug-outline { outline: 2px dashed red; }`. And then I write an HTML template using that utility class, like `<div class="debug-outline">test</div>`, in my Lit element's `render()` method.

This doesn't work. I can see the element's shadow DOM includes the `class="debug-outline"`, and I can see that the page's global CSS includes the `.debug-outline {...}` style rule, but the style doesn't get applied.

As a workaround, I can duplicate the CSS rule into every element where I need it, but this quickly becomes unmaintainable if you have a lot of global utility classes, like if you're using TailwindCSS.

I'm guessing this is just part of how shadow DOM is designed to work. But what about when I actually want certain global CSS utility classes to be available to all my components? Is there some way to do this? Or am I thinking about it wrong?

**Comment by dbwodlf3** [comment]
```html
<link rel="stylesheet" href="/static/index.css" crossorigin="anonymous" />
<my-element>
      #shadow-root
      <link rel="stylesheet" href="/static/index.css" crossorigin="anonymous" />
</my-element>
```

or just you can expose element root. https://lit.dev/docs/components/shadow-dom/#implementing-createrenderroot

```ts

/* ... */

  createRenderRoot(){
    return this;
  }

/* ... */
```
